### PR TITLE
Add sync command option

### PR DESCRIPTION
**Pull request by benoitf**
Referencing https://github.com/eclipse/che/pull/4001

### CODE REVIEW

File: src/main/_docs/setup/setup-cli.md [hunk 1]
```diff
@@ -149,6 +149,12 @@ Connects the current terminal where the command is started to the terminal of a
 The syntax is `ssh <workspace-name> [machine-name]`
 The ssh connection will work only if there is a workspace ssh key setup. A default ssh key is automatically generated when a workspace is created.
 
+### sync
+Synchronize `/assembly` folder with the workspace folder.
```
Comment (JamesDrummond) [inline src/main/_docs/setup/setup-cli.md]: assembly folder? Maybe `Synchronize a local folder with a workspace machine folder`?

Also can we add `Refer to [local ide sync]({{base}}{{site.links["ide-sync"]}}) for additional information.`

**Comment by bmicklea** [comment]
@TylerJewell can you review when you have a moment?

File: src/main/_docs/setup/setup-cli.md (hunk 1 above)
Comment (TylerJewell) [inline src/main/_docs/setup/setup-cli.md]: Should be, "Synchronizes a workspace's contents with a local folder mounted to `:/sync`".

**Comment by benoitf** [comment]
@TylerJewell the doc was updated, it's ok now ?

**Pull request merged**